src/analysis/bounds.rs

```diff
@@ -355,19 +359,21 @@ fn format_out_parameters(parameters: &[String]) -> String {
     }
 }
 
-fn find_error_type(env: &Env, function: &Function) -> String {
+fn find_error_type(env: &Env, function: &Function) -> Option<String> {
     let error_param = function
         .parameters
         .iter()
-        .find(|param| param.direction == ParameterDirection::Out && param.name == "error")
-        .expect("error type");
+        .find(|param| param.direction == ParameterDirection::Out && param.name == "error")?;
```

Comment:
Wat. How does this not fail with any existing code.

This should really check for `throws=1` and then find an out parameter of type `GError**`. I'm sure you can find some examples that use `err` instead of `error` for the parameter name :)

Reply:
Fixed